Legacy Docs Review: Updating delayed retries documentation (#7112)
Particular/docs.particular.net: nservicebus/recoverability/configure-delayed-retries.md

title: Configure delayed retries
summary: How to configure delayed retries as a second stage of recoverability.
component: Core
reviewed: 2025-04-28
redirects:
 - nservicebus/second-level-retries
related:
 - samples/faulttolerance
 - samples/custom-recoverability
---

> [!WARNING]
> Delayed retries cannot be used when transport transactions are disabled or delayed delivery is not available. For more information about transport transactions, refer to [transport transactions](/transports/transactions.md). For more details on delayed delivery, see the [delayed delivery](/nservicebus/messaging/delayed-delivery.md) article.

## Configuring delayed retries

>  * `TimeIncrease`: Specifies the delay interval for each retry attempt. This delay increases by the same timespan with each delayed delivery. The default value is 10 seconds. For example, if the specified value is the default 10 seconds, i.e. 00:00:10, then the first delayed retry will be at ten seconds, the subsequent delayed retry will be 20 seconds, and so on.

Reviewer: ```suggestion
 * `TimeIncrease`: Specifies the delay interval for each retry attempt. This delay increases by the same timespan with each delayed delivery. The default value is 10 seconds. For example, with the default value of 10 seconds, i.e., 00:00:10, the first delayed retry will occur at 10 seconds, the subsequent delayed retry will occur at 20 seconds, and so on.
```
